src/components/GamePage/GamePage.tsx

```diff
@@ -326,7 +326,7 @@ export default function GamePage() {
   }
 
   function getInstallLabel(isInstalled: boolean): React.ReactNode {
-    const { eta, percent } = progress
+    const { eta, bytes, percent } = progress
```

Comment:
So, the thing here is that the information we get from Legendary doesn't really mean that they are the real size. For instance, if you have BArony on your library, this app is just 200MB but Legendary shows 4000MB, I don't know if this is a conversion issue or something else. But Heroic used to have this info until someone points out that those are not the real download size.

So, if you want, you could test more or reach out the guys from Legendary to understand what these number really means :)

Reply:
https://github.com/derrod/legendary/blob/master/legendary/downloader/manager.py#L710

Shouldn't be too difficult to reimplement.

Reply:
Done and dusted.